### ISSUE TITLE
Add SIXEL support

### ISSUE THREAD

**Issue by davidak** [elementary/terminal#717]
### Problem

>The SIXEL format allows the terminal to display bitmap graphics. See https://en.wikipedia.org/wiki/Sixel for more information.

Some programs use it, like Kitten: https://mastodon.ar.al/@aral/110117697614227266

I tested if it works and it seem not.

```
[davidak@gaming:~]$ snow run lsix
Error: Your terminal does not report having sixel graphics support.

Please use a sixel capable terminal, such as xterm -ti vt340, or
ask your terminal manufacturer to add sixel support.

You may test your terminal by viewing a single image, like so:

convert  foo.jpg  -geometry 800x480  sixel:-

If your terminal actually does support sixel, please file a bug
report at http://github.com/hackerb9/lsix/issues

Please mention device attribute codes: ^[[?65;1;9c

[davidak@gaming:~]$ snow run imagemagick -- convert '/home/davidak/Downloads/fdce9e0c0e9ef948.jpg' -geometry 800x480 sixel:-

```

### Proposal

It might be a nice feature for some people. I do not miss it tho.

When done, add an entry to https://www.arewesixelyet.com/, so people know.

Upstream issues for vte:

https://gitlab.gnome.org/GNOME/vte/-/issues/2084
https://gitlab.gnome.org/GNOME/vte/-/issues/253

So you can just wait for upstream to implement it or help them.

### Prior Art (Optional)

See https://www.arewesixelyet.com/

Some terminals with support:

https://github.com/contour-terminal/contour/issues/116
https://codeberg.org/dnkl/foot/releases/tag/1.2.0

**Comment by jeremypw** [comment]
Closing as the fix requires Vte to implement sixel support.

**Comment by rasmus91** [comment]
What, exactly, is this blocked by?

I do almost all my coding in [Helix Editor](https://helix-editor.com/) usually inside a pane in [zellij](https://zellij.dev/) which provides a lean, efficient and nice development environment.

One of the few things this setup is missing is the ability to render graphics, for instance, when creating a mermaid.js diagram, and wanting to view it without opening an imageviewer or browser and continually refreshing. 

The elementary OS terminal is generally the nicest terminal emulator I've used, and to me, at least, this is the one thing it's missing.

**Comment by jeremypw** [comment]
@rasmus91 Adding new features has been held up while working on the port to Gtk4.  That is now finally ready to release so it should be possible to start work on new features soon.

**Comment by jeremypw** [comment]
@rasmus91 I do not use SIXEL graphics myself, so are you aware of any reason why the terminal should have an *option* to enable SIXEL graphics rather than just always have it enabled?

**Comment by jeremypw** [comment]
@rasmus91 I tried enabling SIXEL support in the Vte terminal embedded in Terminal, but it appears not to work although the widget has an `enable-sixel` property.  It seems that support for sixel, having been added is now being discontinued again (see https://github.com/void-linux/void-packages/issues/50220) because it never worked reliably.

SIXEL is a very old format and it is now recommended to move to a more modern format (but there is none yet supported in Vte unfortunately).

I think I am going to have to mark this as "Out of Scope" I am afraid.
